modules/test/playwright/pages/account-admin-web/AccountRolesPage.ts

```diff
@@ -6,10 +6,16 @@
 import {Locator, Page} from '@playwright/test';
 
 export class AccountRolesPage {
+	readonly addNewRoleButton: Locator;
+	readonly editRoleButton: Locator;
 	readonly page: Page;
+	readonly searchInput: Locator;
 
 	constructor(page: Page) {
+		this.addNewRoleButton = page.getByTestId('creationMenuNewButton');
+		this.editRoleButton = page.locator('svg.lexicon-icon-ellipsis-v');
```

Comment:
there is really no other way to get this locator?

Reply:
@smottal Unfortunately, no. I've checked it multiple times, there isn't really any other (more suitable) alternative. And they are also doing the same in poshi.